charts/rode/values.yaml

```diff
@@ -58,6 +58,20 @@ service:
   type: ClusterIP
   port: 50051
 
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+  # kubernetes.io/tls-acme: "true"
+  hosts:
```

Comment:
Nitpick, but I think the example hosts should be commented out as well, since it'll be replaced anyway when using the chart.

Reply:
Is that change what you had in mind? I did the same for the `rode` example, since that's where I copied this from.

Reply:
Yeah that was all I had in mind. Thanks for cleaning up the other default too.